data/data.go

```diff
@@ -4,13 +4,25 @@ type Board [5][5]Holder
 
 // Returns a new 2D board
 func NewBoard() *Board {
-	return nil
+	board := Board{}
+	return &board
 }
 
 // FindNumber finds the given number in the board and returns
 // the row and column of the number alongwith true if value was found
 // else returns (-1, -1, false)
 func (b *Board) FindNumber(num int) (int, int, bool) {
+	i := 0
+	for i < 5 {
```

Comment:
`for i := 0; i < 5; i++` is a better approach

Reply:
Ohh sweet. Wasn't aware the cpp loop syntax works in Go too!